docs/download.md

```diff
@@ -0,0 +1,21 @@
+---
+id: download
+title: Apache Fury(incubating) Download
+---
+
+The official Apache Fury releases are provided as source artifacts.
+
+For binary install, please see Fury [install](/docs/start/install/) document.
+
+## The latest release 
+
+Apache Fury hasn't make a release since joined Apache Incubator.
```

Comment:
There are some grammatical issues - can I suggest?

Apache Fury (Incubating) hasn't made a release since joining the Apache Incubator.

Reply:
Updated, thanks for pointing out those syntax error